reactioncommerce/reaction
# Fix for 3502 shopify sync

Closes #3502 

This adds a new form field to enter the domain to which the user wants the webhook's requests to go to.

### To test
1. Add Shopify credentials in connector
1. Enable "Update Inventory when new Shopify order is placed"
1. Enter the domain that you want the webhook's request to go to(eg - https://reactioncommerce.com/)
1. Click on Setup Sync, sync should be setup successfully.
1. Click `Stop Sync`
1. Notice the previous domain you entered is still pre-filled.
